fdbserver/include/fdbserver/TagPartitionedLogSystem.actor.h

```diff
@@ -212,6 +213,13 @@ struct TagPartitionedLogSystem final : ILogSystem, ReferenceCounted<TagPartition
 	                     Optional<UID> debugID,
 	                     Optional<std::unordered_map<uint16_t, Version>> tpcvMap) final;
 
+	void maybeAdjustBestServer(int bestSet,
```

Comment:
Please note that this API may need to be called in other peek calls too, thanks!

Reply:
The log server availability check currently done in LogSystemPeekCursor.actor.cpp (so we can avoid the need to call this logic explicitly on peek calls). Please check https://github.com/apple/foundationdb/pull/12188 for details, thanks!